portals/ghrc/config.json

```diff
@@ -0,0 +1,20 @@
+{
+    "hasStyles": true,
+    "hasScrpits": false,
+    "logo":{
+        "id": "ghrc-logo",
+        "link": "https://ghrc.nsstc.nasa.gov/home",
+        "tittle": "GHRC Home"
```

Comment:
Typo here, should be `title`

Reply:
Changes made in the latest commit "Fix discovered typos"